smartsim/_core/utils/helpers.py

```diff
@@ -43,18 +43,35 @@
 from datetime import datetime
 from shutil import which
 
+from typing_extensions import TypeAlias
+
 if t.TYPE_CHECKING:
     from types import FrameType
 
     from typing_extensions import TypeVarTuple, Unpack
 
+    from smartsim.launchable.job import Job
+
     _Ts = TypeVarTuple("_Ts")
 
 
 _T = t.TypeVar("_T")
 _HashableT = t.TypeVar("_HashableT", bound=t.Hashable)
 _TSignalHandlerFn = t.Callable[[int, t.Optional["FrameType"]], object]
 
+_nested_seq: TypeAlias = "t.Sequence[Job | _nested_seq]"
```

Comment:
Naming Nit: Traditionally type alias follow the same naming rules as classes (e.g. PascalCase) and should end with `Type`. I think `pylint` will complain about this name if/when we turn that back on. Can we change the name to:
```suggestion
_NestedJobSequenceType: TypeAlias = "t.Sequence[Job | _NestedJobSequenceType]"
```
?